pkgs/desktops/cinnamon/muffin/default.nix

```
@@ -0,0 +1,113 @@
+{ fetchFromGitHub
+, cinnamon-desktop
+, glib
+, file
+, gnome3
+, gnome-doc-utils
+, fetchpatch
+, gobject-introspection
+, gtk3
+, intltool
+, json-glib
+, libinput
+, libstartup_notification
+, libXtst
+, libxkbcommon
+, pkgconfig
+, stdenv
+, udev
+, xorg
+, wrapGAppsHook
+, pango
+, cairo
+, gtk-doc
+, docbook_xsl
+, docbook_xml_dtd_43
+, docbook_xml_dtd_42
+, docbook_xml_dtd_412
+, autoconf
+, automake
+, gettext
+, libtool
+}:
+
+# it's a frankensteins monster with some cinnamon sparkles added on top of it
+
+stdenv.mkDerivation rec {
+  pname = "muffin";
+  version = "4.4.2";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "1kzjw4a5p69j8x55vpbpn6gy8pkbbyii6kzw2nzbypmipgnnijw8";
+  };
+
+  patches = [
+    # backport patch that disables wayland components via build flags
+    (fetchpatch { # https://github.com/linuxmint/muffin/pull/548#issuecomment-578316820
```

The comment pattern was slightly weird.

```suggestion
    # backport patch that disables wayland components via build flags
    # https://github.com/linuxmint/muffin/pull/548#issuecomment-578316820
    (fetchpatch {
```